proto/contlcycle/contlcycle.proto

```diff
@@ -0,0 +1,20 @@
+syntax = "proto3";
+
+option go_package = "github.com/DataDog/agent-payload/v5/contlcycle";
+
+package datadog.contlcycle;
+
+message EventsPaylaod {
+    string version = 1;
+    string eventType = 2;
```

Comment:
does the eventType should be an `enum`? 

Also, IMO eventType should be inside the `message Event`.
Because else it seams that you need to create 2 EventsPayload if you want to send for example `terminated/stop`  and `created` at the same time.

Reply:
+1 for the enum.

About where to put the `eventType`, putting it in the root of the payload would make batching by event type possible if we needed it in the future. Potential consumers could be interested only in few event types.
No strong opinion though.

Reply:
but in that case it means again more message. like 4 messages:
* Pod creation
* Pod termination
* Container creation
* Container termination

IMO the initial idea what to have a stream of event, and user take the one they are interesting on. 
If we want to separated completely the event by type, we should create several `tracks`.

Reply:
Right, I'll move it 👍